we only build manpages here; remove sphinx extensions, not currently in
ports, needed for API docs but not manpages

Index: doc/source/conf.py
--- doc/source/conf.py.orig
+++ doc/source/conf.py
@@ -48,11 +48,9 @@ del update_template_file
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'autoapi.extension',
     'sphinx.ext.autodoc',
     'sphinx.ext.autosectionlabel',
     'sphinx.ext.todo',
-    'sphinx_autodoc_typehints',
 ]
 
 autoapi_type = 'python'
